rabbitmq/metadata.csv

```diff
@@ -29,9 +29,9 @@ rabbitmq.queue.active_consumers,gauge,,,,"Number of active consumers, consumers
 rabbitmq.queue.bindings.count,gauge,,,,Number of bindings for a specific queue,0,rabbitmq,queue bind,
 rabbitmq.queue.consumers,gauge,,,,Number of consumers,0,rabbitmq,cons,
 rabbitmq.queue.consumer_utilisation,gauge,,fraction,,The ratio of time that a queue's consumers can take new messages,0,rabbitmq,cons util,
-rabbitmq.queue.head_message_timestamp,gauge,,millisecond,,"[Management CLI] Timestamp of the head message of the queue [OpenMetricsV2] Timestamp of the first message in the queue, if any",0,rabbitmq,head msg ts,
```

Comment:
Here and elsewhere: please keep the `[OpenMetrics]` marker, you can drop the `v2`. Please also revert the change in the docs where you drop the reference to metrics labeled with this marker. 

We'd like to help customers differentiate between the two versions of this check, and the markers indicate clearly which metrics differ between them.